metadata.json

```diff
@@ -1,87 +1,67 @@
 {
-  "name": "google-glogging",
-  "version": "0.1.3",
-  "author": "Google",
-  "summary": "A Puppet module to manage Google Stackdriver Logging Agent",
+  "name": "ops-cloud_ops",
```

Comment:
```suggestion
  "name": "google-cloud-ops-agents-puppet",
```

Reply:
I went with `google-cloud_ops_agents`, I believe this field is related to Puppet Forge <username>-<module name>. Module name can contain underscores. 

Or should we continue with the convention used in the existing metadata? https://github.com/GoogleCloudPlatform/puppet-google-logging/blob/master/metadata.json